cypress/integration/T3.js

```diff
@@ -0,0 +1,38 @@
+import TaskPage from '../objects/taskPage';
+import Common from '../objects/common';
+
+const taskPage = new TaskPage();
+const common = new Common();
+
+describe('Task creation', function() {
+
+    beforeEach(function () {
+        cy.loginAs('Admin');
+        common.visit('/Tasks');
+    })
+
+    it('Admin creates a new task', function () {
+        
+        common.clickCertainButton('button','Create Task');
+        const taskName = Date.now().toString(); 
+        taskPage.inputIntoASpecificField('taskDetailsForm.name', taskName);
+        taskPage.inputIntoASpecificField('taskDetailsForm.description', Date.now());
+        taskPage.clickBillToClientDropDown();
+        const selection = taskPage.generateRandomBillToClientSelection();
+        taskPage.selectRandomBillToClientOption(selection);
+
+        if (selection === 'Yes') {
+            common.clearField('taskDetailsForm.rate');
+            taskPage.inputIntoASpecificField('taskDetailsForm.rate', Math.floor(Math.random()*50));
+        }
+
+        common.clickCertainButton('Submit','Save');
+
+        common.wait(200);
```

Comment:
explicit waits are evil and should be avoided, better off assert for the actual url, or for some kind of page title (page loaded) and then assert for common.checkIfUrlDoesNotInclude('/create');